ui/src/app/app.component.ts

```diff
@@ -18,7 +18,8 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'atlasmap-dev-root',
-  template: '<router-outlet></router-outlet>',
```

Comment:
This disables multiple mapping definition support, DataMapperExampleHostComponent is now delivered through angular router https://github.com/atlasmap/atlasmap/blob/master/ui/src/app/app-routing.module.ts#L12

As it has been working for browsers without any URL change, I suspect root cause is in vscode-atlasmap and/or vscode webview
https://github.com/jboss-fuse/vscode-atlasmap

@AbbassFaytaroony do you have chance to look into this problem? I'll also look into.

Reply:
It's getting `Unhandled Navigation Error` in @angular/core when it tries to handle `<router-outlet>` via vscode WebView.

Reply:
This is replacing `<base href=` to the absolute URL, but it still doesn't work
https://github.com/jboss-fuse/vscode-atlasmap/blob/master/src/atlasMapWebView.ts#L95